### PR TITLE
README : Add an example for get selected item

### PR DESCRIPTION
**Pull request by gponty**

### CODE REVIEW

**Comment by gponty** [comment]
Add an example for get selected item

File: README.md [hunk 1]
```diff
@@ -70,6 +70,18 @@ You can also use this feature to set which item should be open when mounted.
...
 
+```html
+<script>
+    let currentKey = null;
+    function handleEvent(event){
+        currentKey = event.detail.key;
+    }
+</script>
+
+<Accordion on:change={handleEvent}>
```
Comment (rsdavis) [inline README.md:81]: So, I actually think that you can use two-way binding and then you don't even need on:change. Something like ..
```
<script>
    let currentKey = null
</script>

<Accordion bind:key={currentKey} />
```
Can you check to see if this also works for your use-case?

**Review by rsdavis — CHANGES_REQUESTED** [review]
Thanks for fleshing out the docs here

File: README.md (hunk 1 above)
Comment (gponty) [reply README.md:81]: Oh wow, that's work ! sorry for the inconvenience.
I close this.

Comment (rsdavis) [reply README.md:81]: Ok cool. I'm not sure why I initially put to use on:change.

**Pull request closed without merging**